utils/appliance/implementations/ui.py

```diff
@@ -137,9 +137,11 @@ def create_view(self, *args, **kwargs):
 
     def am_i_here(self):
         try:
-            return self.view.is_displayed
-        except (AttributeError, NoSuchElementException):
+            self.view.assert_is_displayed()
```

Comment:
`return self.view.assert_is_displayed()`

I reckon this would also require changing other views to have this method. Or add a fallback here (first try the assert_is_displayed, then is_displayed) ...